src/box/relay.cc

```diff
@@ -887,7 +913,7 @@ relay_subscribe_f(va_list ap)
 	 * message right away to update the replication lag as
 	 * soon as possible.
 	 */
-	relay_send_heartbeat(relay);
+	relay_do_send_heartbeat(relay);
```

Comment:
I don't really like names such as `do_make_action()` and `make_action()`, because `do` doesn't give any idea how these functions are different. Up to you, but I would recommend renames: `relay_send_heartbeat()` -> `relay_send_heartbeat_if_time()` and `relay_do_send_heartbeat()` -> `relay_send_heartbeat()`, or likewise, but just not `do`.